melodic/distribution.yaml

```diff
@@ -6235,6 +6235,22 @@ repositories:
       type: git
       url: https://github.com/NiryoRobotics/niryo_one_ros_simulation.git
       version: master
+  nmc_nlp_lite-release:
+    doc:
+      type: git
+      url: https://github.com/nmcbins/nmc_nlp_lite_ros.git
+      version: master
+    release:
+      packages:
+      - nmc_nlp_lite
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/nmcbins/nmc_nlp_lite-release.git
+      version: 0.0.7-2
+    source:
+      type: git
+      url: https://github.com/nmcbins/nmc_nlp_lite-release.git
```

Comment:
This should point to the source repository, not the release repository.

Reply:
Thank clalancette!  I am working on this --- by "source repository", you mean "upstream repository" which is , in my case, 
https://github.com/nmcbins/nmc_nlp_ros.git

 am I right?  

Thanks again!

Reply:
correct

Reply:
Thank you tfoote!!!

I think I fixed all the issues being pointed out, please take a look and see if there is still something I missed ...

Thanks again! :)